### PR TITLE
Fixed typos and updated spelling wordlist.

### PR DESCRIPTION
**Pull request by florisdenhengst**

### CODE REVIEW

File: docs/spelling_wordlist [hunk 1]
```diff
@@ -111,8 +136,8 @@ contactdata
 ContentType
 contenttypes
 contrib
-coveredby
 Córdoba
+coveredby
```
Comment (bmispelon) [inline docs/spelling_wordlist]: That looks like an actual typo.

File: docs/spelling_wordlist [hunk 2]
```diff
@@ -607,25 +739,45 @@ stylesheet
...
 subtemplate
 subtemplates
+subtree
+subtype
 subviews
 subwidgets
+superclass
+superset
+swappable
+swiss
```
Comment (bmispelon) [inline docs/spelling_wordlist]: should probably be "Swiss" (capital S)

File: docs/spelling_wordlist [hunk 3]
```diff
@@ -650,36 +806,54 @@ timelines
...
 trac
+tracebacks
+transactional
 Transifex
 Tredinnick
 triager
 triagers
+triaging
+Triaging
+trigged
```
Comment (bmispelon) [inline docs/spelling_wordlist]: I don't think that's a word :)

File: docs/spelling_wordlist [hunk 4]
```diff
@@ -742,8 +929,11 @@ webpages
 webserver
 webservers
 whatsnext
+Where'd
```
Comment (bmispelon) [inline docs/spelling_wordlist]: That's probably wrong as well

**Comment by bmispelon** [comment]
The docs build failure seems unrelated.
Merged in e42a720ba22a51c68d2316d425234a197eaff467.

Dankjewel!

**Comment by timgraham** [comment]
Any ideas why those words needed to be added the wordlist? I haven't looked into where Sphinx gets its dictionary from, but there have been reports of similar errors on other systems and it would be nice if we could pinpoint the reason for this and possibly add it as an FAQ.

**Comment by timgraham** [comment]
From IRC: "on the 'wordlist commit': the Sphinx dictionary that is being used is aspell-en_US. The words that were added in the commit were not present in that dictionary, but were used in the documentation. Arch linux."

Another question is if we need to add both lower and uppercased version of the same words.

**Comment by florisdenhengst** [comment]
Adding just upper or lowercase would imply doing a case insensitive check agains the wordlist, which might not be correct for names, acronyms and abbreviations. 
Examples: 
- 'Swiss army knife' vs. 'swiss army knife'
- 'DoS' (Denial of Service) vs. 'DOS' (Disk Operating System) vs. 'dos' (incorrect)

**Pull request closed without merging**